app/code/Magento/Sitemap/Test/Unit/Controller/Adminhtml/Sitemap/SaveTest.php

```diff
@@ -50,8 +49,41 @@ class SaveTest extends \PHPUnit\Framework\TestCase
      */
     protected $messageManagerMock;
 
+    /**
+     * @var \Magento\Framework\Validator\StringLength|\PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $lengthValidator;
```

Comment:
it's preferable to declare properties as private unless they are used by child classes

Reply:
okay sure